conf/tidb.yml

```diff
@@ -44,57 +31,25 @@ log:
   # Log level: debug, info, warn, error, fatal.
   # level: "info"
 
-  # Log format, one of json, text, console.
-  # format: "text"
-
-  # Disable automatic timestamps in output
-  # disable-timestamp: false
-
   # Queries with execution time greater than this value will be logged. (Milliseconds)
   # slow-threshold: 300
 
   # Queries with internal result greater than this value will be logged.
   # expensive-threshold: 10000
 
-  # Maximum query length recorded in log.
```

Comment:
we need to mention these removed config items in our [documents](https://pingcap.com/docs-cn/v3.0/reference/configuration/tidb-server/configuration-file/).

Reply:
OK, I'll check it